server/sonar-web/src/main/js/helpers/handlebars/issueFilterTotalLink.js

```diff
@@ -34,9 +34,9 @@ module.exports = function (query, mode) {
     r.facetMode = 'debt';
   }
   if (r.componentKey != null) {
-    return '/component_issues/index?id=' + encodeURIComponent(r.componentKey) +
+    return window.baseUrl + '/component_issues/index?id=' + encodeURIComponent(r.componentKey) +
         '#' + getQuery(_.omit(r, 'componentKey'));
   } else {
-    return '/issues/search#' + getQuery(r);
+    return window.baseUrl + '/issues/search#' + getQuery(r);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)